ejb3/src/main/java/org/jboss/as/ejb3/remote/EJBRemoteConnectorService.java

```diff
@@ -75,35 +70,14 @@ public void start(StartContext context) throws StartException {
         final AssociationService associationService = associationServiceInjectedValue.getValue();
         final Endpoint endpoint = endpointValue.getValue();
         Executor executor = executorService.getOptionalValue();
-        if(executor != null) {
+        if (executor != null) {
             associationService.setExecutor(executor);
         }
         RemoteEJBService remoteEJBService = RemoteEJBService.create(
             associationService.getAssociation(),
             remotingTransactionServiceInjectedValue.getValue()
         );
-        final ControlledProcessStateService processStateService = controlledProcessStateServiceInjectedValue.getValue();
```

Comment:
Are you sure this is safe?  A service can be `ON_DEMAND` and still start up eagerly.